### PR TITLE
updated the multiplay-module (GUID-error-fix)

### LINKED ISSUE #38: Multiplay component module - GUID error
## Module

* [ ] Module Importer
* [X] Multiplay Component
* [ ] Interaction
* [ ] Gesture
* [ ] Screenshot
* [ ] Leaderboard
* [ ] Mannequin
* [ ] Product
* [ ] Localisation
* [ ] QuickChat

## Version
Write the version that you are currently using.
Multiplay component v1.1.1

## Describe the bug (actual behavior)

Currently when I try to import the Assets such as City or Commons from the Zepeto.Assets in Unity Package Manager, there is a GUID conflicts with the existing assets in the Multiplay Component. Module

### Unity
- Version
  
### Zepeto World SDK
- Version

### PR DESCRIPTION
**Pull request by magmukendi**
In this PR,

- Fixed the GUID error mentioned in the #38 
- Removed custom modules (product module and mannequin module) from the ServerModule list in the index.ts file
- Removed the ProductModule.ts and MannequinModule.ts from the ServerModule folder.
- Added the Zepeto Multiplay Messaging to the index.d.ts file.
- Removed the Resources folder from the Multiplay Component Module ( this folder had no purpose in the Multiplay Module ).
- Removed the SampleScene and the SampleScript from the Multiplay module
- Removed the DemoMovingBlock.unity scene from the Multiplay Component module (this Scene had missing Assets).

Close #38

### CODE REVIEW

**Comment by magmukendi** [comment]
I figured out that the balloons, the cushion, and the ghost material were referencing to `all_colortexture1` material included in the Zepeto.Assets in the commons folder. I changed that so that they reference to the `all_colortexture1` in the `_Resources > Materials` folder included in the Zepeto Multiplay Component folder

**Comment by magmukendi** [comment]
In the new commit, I did the following:

- updated the `all_colortexture1.mat` file by updating the the reference to the `all_colortexture1.png` file I changed it so t hat it now reference to the `all_colortexture.png` located in the _Resources > Materials folder included in the Zepeto Multiplay Component folder. (See image below).
Screen Shot 2023-09-05 at 10 34 24 AM

- moved the Balloon(TransformSync) and Ghost(TransformSync) prefabs from the `Resource` folder outside of the multiplay component to the `_Resources` inside the Multiplay component.

**Review by bohyunjung — CHANGES_REQUESTED** [review]
@magmukendi

TL: DR; The prefabs should be able to be found under /Asset/Resources/ if we're not going to update how Multiplay Manager works.


As of the latest commit, While the ballons and the ghost pillow render as expected, the interaction with those objects fails.

This is because `MultiplayManager.Instantiate` finds the prefabs under /Assets/Resources.

That's why the assets used in the multiplay component demo scene were located under the global asset directory.

Please take a look at the screenshots with the error message below.

스크린샷 2023-09-07 오전 10 58 08

스크린샷 2023-09-07 오전 10 57 49

**Comment by magmukendi** [comment]
@bohyunjung Thanks so much for the review. I should have keep an eye on the behavior of each GameObject (Balloon and Ghost)

I double checked the project again. As the code is using the Resource API I think we do not really need to put the assets related to the Multiplay module in the `Assets/Resources`. Any subfolder named Resources would work.

In this new commit, I made the following changes:
- Renamed the `_Resources` in the Multiplay component to `Resources`.
- changed the string path in the `InstantiateObject.ts` file by adding `Prefabs/*`.

**Comment by magmukendi** [comment]
I just made another commit that does the following:

- Update the AllComponent Scene in the `Assets/Scene` folder so that the Balloon and the cushion are renderer and function properly.
- Update the `Assets/Resources` folder so that there isn't any missing material.
- Update the Multiplay Component v1.1.2

**Review by bohyunjung — APPROVED** [review]
👍
